### PR TITLE
Greedy CC

### PR DESCRIPTION
**Pull request by msluszniak**

### CODE REVIEW

**Comment by msluszniak** [comment]
[Writing a base class/function for calculating greedy algorithms](https://trello.com/c/gwheO6s5/51-writing-a-base-class-function-for-calculating-greedy-algorithms)

**Comment by msluszniak** [comment]
[CC-rule implementation: Greedy algorithm](https://trello.com/c/pOZ070uq/36-cc-rule-implementation-greedy-algorithm)

File: votepy/chamberlin-courant.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        voting (Union[OrdinalElection, list[int]]): Voting for which the function calculates the committee
+        size_of_committee (int): Size of the committee
+        number_of_scored_candidates (int): Number of scored candidartes using k-borda rule
+    Returns:
+        list[int]: List of chosen candidates
+    """
+
+    def scoring_function(committee: Iterable[int], voting: OrdinalElection, candidate: int,
+                         number_of_scored_candidates: int):
+        committee = set(committee + [candidate])
```
Comment (Wolokin) [inline votepy/chamberlin-courant.py]: committee is an Iterable and for most Iterables adding a list won't work. 
```suggestion
        committee = set(committee) | {candidate}
```

**Review by Wolokin — CHANGES_REQUESTED** [review]
Almost fine

**Pull request merged**